qdrant-landing/content/documentation/qdrant-alternatives.md

```diff
@@ -0,0 +1,78 @@
+| **Query**      | **Search**     | Name of the method used to find the nearest neighbors for a given vector, possibly with some additional filters applied on top.                                                                                                                                                |
+| N/A            | **Scroll**     | Pinecone does not offer a way to iterate through all the vectors in a particular index. Qdrant has a `scroll` method to get them all without using search.                                                                                                                     |
+
+## Known limitations
+
+1. Pinecone does not support arbitrary JSON metadata, but a flat structure with strings, numbers, booleans, or lists of strings used as values. Qdrant accepts any JSON object as a payload, even nested structures.
+2. NULL values are not supported in Pinecone metadata but are handled properly by Qdrant.
+3. The maximum size of Pinecone metadata is 40kb per vector. 
+4. Pinecone, unlike Qdrant, does not support geolocation and filtering based on geographical criteria.
+5. Qdrant allows storing multiple vectors per point, and those might be of a different dimensionality. Pinecone doesn't support anything similar.
+6. Vectors in Pinecone are mandatory for each point. Qdrant supports optional vectors.
+
+It is worth mentioning, that **Pinecone will automatically create metadata indexes for all the fields**. Qdrant assumes you know
+your data and your future queries best, so it's up to you to choose the fields to be indexed. Thus, **you need to explicitly define the payload indexes while using Qdrant**.
+
+## Supported technologies
+
+Both tools support various programming languages providing official SDKs.
+
+|                           | Pinecone             | Qdrant               |
+|---------------------------|----------------------|----------------------|
+| **Python**                | ✅                    | ✅                    |
+| **JavaScript/TypeScript** | ✅                    | ✅                    |
+| **Rust**                  | ❌                    | ✅                    |
+| **Go**                    | ❌                    | ✅                    |
+
+There are also various community-driven projects aimed to provide the support for the other languages, but those are not officially 
+maintained, thus not mentioned here. However, it is still possible to interact with both engines through the HTTP REST or gRPC API. 
+That makes it easy to integrate with any technology of your choice.
+
+If you are a Python user, then both tools are well-integrated with the most popular libraries like LangChain, LlamaIndex, Haystack, and more. 
```

Comment:
We should link to these integrations for Qdrant at least

Reply:
done